.circleci/config.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
```

Comment:
Unless this change is necessary for this functionality, please put it in a separate PR.

Reply:
This was the first commit of this PR (03b32d6dfee03aaed5b3759a24fe5ef8d54cd943). I openend #147, containing just this one commit.